server/src/data/measurables-data.js

```diff
@@ -0,0 +1,156 @@
+const Sequelize = require('sequelize');
+
+const models = require("../models");
+const { notify } = require("../lib/notifications");
+
+const { UsersData } = require('./users-data');
+
+class MeasurablesData {
+
+  constructor() {
+    this.usersData = new UsersData();
+  }
+
+  /**
+   * @param root
+   * @param values
+   * @param options
+   * @return {Promise<*>}
+   */
```

Comment:
How useful do you find these annotations? Most seem fairly obvious to me.

Reply:
they help with code completion http://usejsdoc.org
also we can use typescript d.ts files to do better IDE code completion

Reply:
Sounds good. Happy to discuss/move to typescript. I do prefer reasonML though, so am hesitant to change if it would be a lot of work, and we may change again in 9-20 months.